common/types/matching.go

```diff
@@ -455,6 +456,14 @@ func (v *MatchingPollForDecisionTaskResponse) GetBranchToken() (o []byte) {
 	return
 }
 
+// GetHistorySize is an internal getter of returning the history size in bytes
+func (v *MatchingPollForDecisionTaskResponse) GetHistorySize() (o int64) {
```

Comment:
The mismatch between field name and getter name is confusing.

Reply:
Resolved